### DISCUSSION TITLE
Collectives error messages in AIO 3.0.0

### DISCUSSION THREAD

**Discussion by davidszp** [nextcloud/all-in-one#1458] [Questions] [answered]
### Steps to reproduce
1. Fresh install of AIO 3.0.0, after following the "reset the AIO instance properly" guide
2. Install Collectives from Apps menu
3. Open Collectives tab, create an item using the "+" link on left-hand side. 

### Expected behavior 
Item should be created without error message

### Actual behavior 
Error message "Can't load page" and (after attempts to add the item again) "Item already exists"

### Host OS 
Unraid Docker Linux/AMD

#### Nextcloud AIO version 
3.0.0

#### Current channel 
Latest

#### Other valuable info 
Reinstalling Docker nextcloud/all-in-one:20221103_091721-latest fixed the problem, and Collectives works as expected.

**Comment by szaimen** [comment, marked as answer]
I think this should be fixed with NC24.0.8. Alternatively check out https://github.com/nextcloud/all-in-one/discussions/1434
